wof/core_1_1.py

```diff
@@ -589,7 +589,7 @@ def create_source_element(self, sourceResult):
 
         contactInfo = self.create_contact_info_element(sourceResult)
 
-        source.ContactInformation = contactInfo
+        source.contactInformation = [contactInfo]
```

Comment:
@ocefpaf I've already checked. It is expecting a list but for some reason is not. And based on https://github.com/ODM2/WOFpy/blob/master/wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py#L328 Affiliation is passed through only once.